guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -80,7 +80,7 @@ When possible, profile on devices that have similar performance characteristics
 
 #### Choose a lower local video quality
 
-Sometimes it is better to sacrifice video quality in order to prioritize audio. You can call [chooseVideoInputQuality](https://aws.github.io/amazon-chime-sdk-js/interfaces/devicecontroller.html#choosevideoinputquality)(width, height, frameRate, maxBandwidthKbps) and lower the maximum bandwidth in real-time. You can also adjust the resolution and frame rate if you call the method before starting local video (or stop and then restart the local video). See the section below on values you can use for `chooseVideoInputQuality`.
+Sometimes it is better to sacrifice video quality in order to prioritize audio. You can call [chooseVideoInputQuality(width, height, frameRate)](https://aws.github.io/amazon-chime-sdk-js/interfaces/devicecontrollerfacade.html#choosevideoinputquality) and [setVideoMaxBandwidthKbps(maxBandwidthKbps)](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#setvideomaxbandwidthkbps) to lower the maximum bandwidth in real-time. You can also adjust the resolution and frame rate if you call the method before starting local video (or stop and then restart the local video). See the section below on values you can use for `chooseVideoInputQuality`.
```

Comment:
Oops, I missed this.
The link of `setVideoMaxBandwidthKbps` also needs an update. It's on `AudioVideoControllerFacade`:
https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideocontrollerfacade.html#setvideomaxbandwidthkbps

Reply:
I think we are referencing everything from `AudioVideoFacade`, cause technically builder will use these APIs from `AudioVideoFacade` or `DeviceController`, they should not know about `..ControllerFacade`. Searched "https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html" and we do not use internal facades for linking in guides across JS SDK. `DeviceController` makes sense to use though, I checked `startVideoInput` and `startAudioInput` and they reference `AudioVideoFacade`, hence I think it makes sense to use `AudioVideoFacade` reference everywhere.

Reply:
I agree with what you said. My thought is that, for API usage we already have code example below:
```js
// 360p
meetingSession.audioVideo.chooseVideoInputQuality(640, 360, 15);
meetingSession.audioVideo.setVideoMaxBandwidthKbps(600);
// 540p
meetingSession.audioVideo.chooseVideoInputQuality(960, 540, 15);
meetingSession.audioVideo.setVideoMaxBandwidthKbps(1400);
// 720p
meetingSession.audioVideo.chooseVideoInputQuality(1280, 720, 15);
meetingSession.audioVideo.setVideoMaxBandwidthKbps(1400);
```
I think it's good enough to let builder know they can access the API via `meetingSession.audioVideo.*`.

That link is more like API Reference, and it has an anchor `#setvideomaxbandwidthkbps` which does not exist on that page. It feels like we are giving a broken API reference to builder. This is my concern.

Reply:
Discussed offline, link to `AudioVideoFacade` for because we are referencing everything from `AudioVideoFacade`